week3/solutions/RO_Flowsheet_ERD_Techno_Economics.ipynb

```diff
@@ -0,0 +1,1886 @@
+      "\n",
+      "------------------------------------------------------------------------------------\n",
+      "    Stream Table\n",
+      "                                              Units         Feed Inlet  Feed Outlet  Permeate Outlet\n",
+      "    flow_mass_phase_comp ('Liq', 'H2O')  kilogram / second     0.98796     0.48954        0.49842   \n",
+      "    flow_mass_phase_comp ('Liq', 'TDS')  kilogram / second    0.035000    0.034888     0.00011159   \n",
+      "    temperature                                     kelvin      298.15      298.15         298.15   \n",
+      "    pressure                                        pascal  7.5000e+06  6.9676e+06     1.0132e+05   \n",
+      "====================================================================================\n",
+      "\n",
+      "====================================================================================\n",
+      "Unit : fs.erd                                                              Time: 0.0\n",
+      "------------------------------------------------------------------------------------\n",
+      "    Unit Performance\n",
+      "\n",
+      "    Variables: \n",
+      "\n",
+      "    Key             : Value       : Units         : Fixed : Bounds\n",
+      "         Efficiency :     0.95000 : dimensionless :  True : (None, None)\n",
+      "    Mechanical Work :     -3265.5 :          watt : False : (None, None)\n",
+      "    Pressure Change : -6.8663e+06 :        pascal : False : (None, None)\n",
+      "     Pressure Ratio :    0.014542 : dimensionless : False : (None, None)\n",
+      "\n",
+      "------------------------------------------------------------------------------------\n",
+      "    Stream Table\n",
+      "                                              Units           Inlet     Outlet  \n",
+      "    flow_mass_phase_comp ('Liq', 'H2O')  kilogram / second    0.48954    0.48954\n",
+      "    flow_mass_phase_comp ('Liq', 'TDS')  kilogram / second   0.034888   0.034888\n",
+      "    temperature                                     kelvin     298.15     298.15\n",
+      "    pressure                                        pascal 6.9676e+06 1.0132e+05\n",
+      "====================================================================================\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Report unit-level performance results for (1) the pump, (2) RO and (3) the energy recovery device, using the report() method for each unit on the flowsheet.\n",
+    "\n",
+    "m.fs.pump.report()\n",
+    "m.fs.RO.report()\n",
+    "m.fs.erd.report()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "774c11d9",
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  },
+  {
+   "cell_type": "markdown",
+   "id": "aa31b30d-46e7-4ea6-80bc-a509304a68a9",
+   "metadata": {},
+   "source": [
+    "## Step 7: Add Costing"
```

Comment:
Is it worth combining all of the steps up til this point into a few steps? Seems like a waste of class time to go through all the steps of flowsheet construction again